src/test/java/com/snowflake/kafka/connector/streaming/iceberg/IcebergIngestionSchemaEvolutionIT.java

```diff
@@ -483,4 +483,15 @@ private static Stream<Arguments> testEvolutionOfComplexTypes_dataSource() {
             twoObjectsExtendedWithMapAndArrayPayload(),
             false));
   }
+
+  @Test
+  @Disabled
+  void shouldAppendedCommentTest() throws Exception {
```

Comment:
Could you test adding the default comment in any of the tests?

Reply:
I can.

Reply:
done